packages/language-server/src/quick-fix.ts

```diff
@@ -0,0 +1,103 @@
+import { ExecuteCommandParams } from "vscode-languageserver";
+import {
+  Diagnostic,
+  Range as LSPRange,
+  TextDocument,
+  CodeAction,
+  Command,
+  CodeActionKind,
+  TextDocumentEdit,
+  TextEdit,
+} from "vscode-languageserver-types";
+import { getQuickFixIdInfo } from "@ui5-language-assistant/xml-views-quick-fix";
+import { parse, DocumentCstNode } from "@xml-tools/parser";
+import { buildAst, XMLDocument } from "@xml-tools/ast";
+import { LSPRangeToOffsetRange, offsetRangeToLSPRange } from "./range-utils";
+
+export function getQuickFixCodeAction(
```

Comment:
Can we decouple this form `vscode-languageserver-types`
What if someone wants to use this quick-fix package in implementing some other language-server?